StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/testing.mon

```diff
@@ -0,0 +1,27 @@
+monitor Testing {
+	
+	
+	event Trigger {}
+	action onload() {
+		spawn testCode();
+	}
+	
+	float counter := 100.0;
+	
+	action testCode() {
+		monitor.subscribe("out");
+
+		
+		on all Trigger() {
+			float reading := counter.rand();
+			log "Sending "+reading.toString() ;
+			send Temperature("Sensor1", reading) to "in";
+			
+			on wait(1.0) {
+				send Trigger() to context.current();
+			}
+		}
+		route Trigger();
+		
+	}
+}
```

Comment:
Missing newline at end of file.
(in general this is good practice for most plain text files under version control)

Reply:
Yeah understood , but thought of removing this file since this testing.mon I created to test it on my local dev machine which anyways i am achieving through python script uploaded, so removed this file.